### ISSUE TITLE
Dark theme error color contrast issues

### ISSUE THREAD

**Issue by robsturgill-tyler** [tyler-technologies-oss/forge#648]
**Describe the bug:**

Early dev of 3.x the had the CSS var `--forge-theme-error` color value of `#ec8396` or `$ruddy-pink-200` it's currently using `#CF0038` or `$ruddy-pink-500`. This color change has cascaded through a bunch of components like badges, chips (invalid), etc which now fail a11y contrast in dark theme. Teams [Forge > General thread](https://teams.microsoft.com/l/message/19:1gCw-wLjrS56X_iAY0mpHpk4PyWOi08JiMM7Kc1xvho1@thread.tacv2/1720536021757?tenantId=7cc5f0f9-ee5b-4106-a62d-1b9f7be46118&groupId=2bb1b8ea-cd12-4df5-a4ab-0c895bfcda5c&parentMessageId=1720536021757&teamName=Forge&channelName=General&createdTime=1720536021757)

**Expected behavior:**

Change the value in the [_tokens.status.scss](https://github.com/tyler-technologies-oss/forge/blob/acbc9902e636e69771b41cdc0ad4fce453811f8e/src/lib/core/styles/tokens/theme/_tokens.status.scss#L60) to `$ruddy-pink-200` outputting to `#ec8396`. Verify across components using error theme/states

**Screenshots:**

![error-color](https://github.com/tyler-technologies-oss/forge/assets/74620886/71415458-a228-4248-87d4-766e1260f42e)

**Please complete the following information:**
 - Forge version: V3.0
 - I have searched existing issues before creating this report? Y
 - Browser: All
 - Platform: All
 - OS: All

**Additional context:**

This is a global theme change so it will likely majority of components including state overlays like hover.
